api/nakadi-event-bus-api.yaml

```diff
@@ -1634,6 +1670,37 @@ definitions:
       purposes. We recommend logging this object in order to solve connection issues. Clients
       should not parse this structure.
 
+  CursorDistanceQuery:
+    required:
+      - partition
+      - initial_offset
+      - final_offset
```

Comment:
another option would be for `CursorDistanceQuery` to consist of `initial_cursor` and `final_cursor`, both of type `Cursor`. this would introduce a tiny amount of duplication (same `partition`), but might be more user friendly?